README.md

```diff
@@ -153,6 +180,28 @@ client.Ethereum.listRewards(filter).then((resp) => {
 
    </details>
 
+## Build
```

Comment:
Maybe we can make a `Contributing.md` and link that doc here? That way, someone who is trying to build/contribute to this codebase can refer to a different file than someone trying to use it. My preference would be to keep the top-level README as concise as possible.

Reply:
Yes, lets do that in a separate PR